webpack.config.babel.js

```diff
@@ -107,6 +107,7 @@ const config = {
       $: 'jquery',
       jQuery: 'jquery',
       'window.jQuery': 'jquery',
+      katex: 'katex',
```

Comment:
`auto-render.js` directly referenced a global `katex`, instead of importing it by `require('katex')`.